libs/scripts/scripts.js

```diff
@@ -109,6 +109,7 @@ const config = {
   fallbackRouting: 'on',
   links: 'on',
   imsClientId: 'milo',
+  imsScope: 'AdobeID,openid,gnav',
```

Comment:
I wish there was a better place to put this stuff. In five years, the config object is going to be huge.